redwood-layout/README.md

```diff
@@ -0,0 +1,102 @@
+Redwood Layout
+=======
+
+This artifact includes `Row` and `Column` widget implementations for Android Views, Android Compose UI, and iOS UIKit.
+
+To being, add the follow entries to your `schema.kt`:
+
+```kotlin
+@Widget(1)
+data class Row(
+  @Property(1) @Default("Padding.Zero") val padding: Padding,
+  @Property(2) @Default("Overflow.Clip") val overflow: Overflow,
+  @Property(3) @Default("MainAxisAlignment.Start") val horizontalAlignment: MainAxisAlignment,
+  @Property(4) @Default("CrossAxisAlignment.Start") val verticalAlignment: CrossAxisAlignment,
+  @Children(5) val children: RowScope.() -> Unit,
+)
+
+object RowScope {
+  /** https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow */
+  fun LayoutModifier.grow(value: Int): LayoutModifier {
+    return then(GrowLayoutModifier(value))
+  }
```

Comment:
In the schema layout modifiers are defined like widgets and scopes are empty types.

So this should look something like
```kotlin
object RowScope

@LayoutModifier(1, RowScope::class)
data class Grow(
  val value: Int,
)
```

Reply:
@JakeWharton I think we might need to define these manually since they can't be generated or the implementation classes can't reference those modifiers. Will it cause issues if those don't have an associated tag?

Reply:
@JakeWharton I like that idea - let's do it! I'm guessing `Flexbox` in this example is a reference to the child `@Schema`-annotated interface? This would also let me get rid of the `Bridge` classes in the `README`.

If it's OK I'll also leave the layout modifiers as is for the moment so I can reference them in the implementation PR. I'll update it to the correct way once it's possible to add dependencies.

Reply:
Sounds good. I'm out tomorrow so this'll be a next week thing.